tools/fonts/inspector/otf.html

```diff
@@ -214,7 +214,7 @@
         await gonville.load();
         document.fonts.add(gonville);
         const robotoSlabOTF = '../@/robotoslab/RobotoSlab-Medium_2.001.otf';
-        const robotoSlab = new FontFace('Roboto Slab', `url(${robotoSlabOTF}) format('opentype')`);
+        const robotoSlab = new FontFace('RobotoSlab', `url(${robotoSlabOTF}) format('opentype')`);
```

Comment:
Just checking if we are using spaces in all the font names. Is it supposed to be "Roboto Slab", with a space?

Reply:
done